app/backend/src/clj/teet/integration/vektorio/vektorio_client.clj

```diff
@@ -37,14 +38,15 @@
                         "Content-Type" "application/json"}
                        headers)
         content-type (get headers "Content-Type")
-        resp @(http/post (str api-url endpoint)
-                           {:headers headers
-                            :body (if (= content-type "application/octet-stream")
-                                    payload
-                                    (cheshire/encode payload))})]
+        resp (try (clj-http/post (str api-url endpoint)
+                                 {:headers headers
+                                  :body (if (= content-type "application/octet-stream")
+                                          payload
+                                          (cheshire/encode payload))})
+                  (catch Exception e
+                    (log/fatal e "Exception in vektorio post for url " (str api-url endpoint))))]
```

Comment:
what will `vektor-message-handler` do in the catch case, resp will be `nil`. Should the exception be rethrown to stop it from trying to handle nil